packages/http/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @turnkey/http
 
+## 2.12.3
+
+### Patch Changes
+
+- e2f2e0b: updates to sdk-browser, sdk-server and http
```

Comment:
We can mention just the changes to `http` here (pulled from https://github.com/tkhq/sdk/pull/341 -- primarily two new endpoints, for deleting private keys + wallets)

We don't have to mention `sdk-browser` and `sdk-server` here given `http` does not depend on those libraries, and in fact the opposite is the case (those libraries depend on `http`)